src/web/components/Onwards/OnwardsContainer.tsx

```diff
@@ -6,9 +6,14 @@ import { from } from '@guardian/src-foundations/mq';
 type Props = {
     children: JSX.Element | JSX.Element[];
     dataComponentName: string;
+    dataLinkName: string;
```

Comment:
Could this be 'linkName'?

I know I'm coming at this without any experience of this area so maybe the data prefix is established but it seems like `data-` is just how the browser identifies the attribute as different and isn't a true description of the data itself (hope that makes sense!)

Reply:
Yeah, you're probably right, however it's used 462 times throughout the codebase so probably best to stick as is for now and batch replace if we think it's an issue! :)

Reply:
Yikes. Okay then... let's back slowly away from that one